transformation/src/test/scala/org/broadinstitute/monster/hca/HcaPipelineBuilderSpec.scala

```diff
@@ -118,6 +118,35 @@ class HcaPipelineBuilderSpec extends AnyFlatSpec with Matchers {
     actualOutput shouldBe expectedOutput
   }
 
+  it should "transform links.json file" in {
+    val exampleMetadataContent = JsonParser.parseEncodedJson(
+      json = """
+               | {
+               |    "schema_type": "link_bundle",
+               |    "schema_version": "1.1.3",
+               |    "links": []
+               | }
+               |""".stripMargin
+    )
+    val actualOutput = HcaPipelineBuilder.transformLinksFileMetadata(
+      filename = "123_456_789.json",
```

Comment:
One of the actual filenames I grabbed (and changed a few numbers on) looks like this: 
`a002c1c6-eb37-4463-bd30-cf738e2a435f_2019-07-01T200157.523509Z.json` 

I think they only have one underscore in them, unless something changed.

Reply:
Oh, it did change. This looks good :)